RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3823,7 +3823,7 @@ boolean buffMaintain(effect buff, int mp_min, int casts, int turns, boolean spec
 	case $effect[Eldritch Alignment]:			useItem = $item[Eldritch Alignment Spray];		break;
 	case $effect[Elemental Saucesphere]:		useSkill = $skill[Elemental Saucesphere];		break;
 	case $effect[Empathy]:
-		if(have_familiar($familiar[Mosquito]) && acquireTotem())
+		if(pathAllowsFamiliar() && acquireTotem())
```

Comment:
These should be the very last places where use a check fo the existence of the Mosquito familiar as a proxy for being able to use familiars.